lnwallet/log.go

```diff
@@ -29,7 +35,7 @@ func DisableLog() {
 func UseLogger(logger btclog.Logger) {
 	walletLog = logger
 
-	btcwallet.UseLogger(logger)
+	//btcwallet.UseLogger(logger)
```

Comment:
Stray comment.

Reply:
Fixed, also changed `DisableLog()` to call `UseLogger` to ensure the sublogger is propogated to dependent packages